zello-sdk-sample-misc/src/main/java/com/zello/sdk/sample/misc/MiscActivity.java

```diff
@@ -144,29 +140,32 @@ public boolean onPrepareOptionsMenu(Menu menu) {
 
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item) {
-		switch (item.getItemId()) {
-			case R.id.menu_available:
-			case R.id.menu_solo:
-			case R.id.menu_busy:
-				chooseStatus();
-				break;
-			case R.id.menu_enable_auto_run:
-				Zello.getInstance().setAutoRun(true);
-				break;
-			case R.id.menu_disable_auto_run:
-				Zello.getInstance().setAutoRun(false);
-				break;
-			case R.id.menu_enable_auto_connect_channels:
-				Zello.getInstance().setAutoConnectChannels(true);
-				break;
-			case R.id.menu_disable_auto_connect_channels:
-				Zello.getInstance().setAutoConnectChannels(false);
-				break;
-			case R.id.menu_open_app:
-				Zello.getInstance().openMainScreen();
-				break;
+		int id = item.getItemId();
+		if (id == R.id.menu_available || id == R.id.menu_solo || id == R.id.menu_busy) {
```

Comment:
I'm assuming there was a good reason to change these from `switch` to `if` blocks, but I think I'm missing something. Can you clarify?

Reply:
Yes – they won't compile by default anymore. The resource IDs are no longer constants by default, and I did not want to modify AGP's behavior to make them constant again.